maps/away/ships/freebooter/freebooter_salvager/freebooter_salvager.dm

```diff
@@ -0,0 +1,150 @@
+	class = "ICV"
+	icon_state = "tramp"
+	moving_state = "tramp_moving"
+	color = "#a0a8ec"
+	max_speed = 1/(2 SECONDS)
+	burn_delay = 1 SECONDS
+	vessel_mass = 10000 // very inefficient in terms of thrust
+	fore_dir = SOUTH
+	vessel_size = SHIP_SIZE_SMALL
+	invisible_until_ghostrole_spawn = TRUE
+	designer = "Einstein Engines"
+	volume = "56 meters length, 39 meters beam/width, 22 meters vertical height"
+	drive = "Low-Speed Warp Acceleration FTL Drive"
+	weapons = "Dual improvised weapon arrays, underside shuttle docking compartment."
+	sizeclass = "Tartarus-class Bulk Hauler"
+	shiptype = "Long-term shipping utilities"
+	initial_restricted_waypoints = list(
+		"Freebooter Salvager Shuttle" = list("fbs_nav_hangar")
+	)
+	initial_generic_waypoints = list(
+		"fbs_nav1",
+		"fbs_nav2",
+		"fbs_nav3",
+		"fbs_nav4",
+		"fbs_nav5",
+		"fbs_nav6",
+		"fbs_nav7",
+		"fbs_nav8",
+		"fbs_upfore"
+	)
+
+// copy-pasted from original freebooters
+/obj/effect/overmap/visitable/ship/freebooter_salvager/New()
+	designation = "[pick("Peaceful", "Charming", "Jackal", "Friendly", "Boar", "One and Done", "Nothing Suspicious", "Death Valley", "Scan Again", "Pickpocket", "Dashing Rogue", "Port Authority", "Customs Patrol Ship", "Pay Transit Tolls Here", "Immigration Inspector", "For Hire", "Ex-Military", "Eat Shit", "Fuck Off", "Big Brother", "Wrong Turn", "Claim Jumper", "Davy Jones", "Thenardier", "Killer Queen", "Gunpowder", "Dynamite", "Phoron Collection Service", "We Never Liked Working Anyway", "The Orphanage", "Undertaker", "Flip and Burn", "Emphasis on Independent", "Widowmaker", "Irregular", "Ned Kelly", "Pay Your Union Dues Here", "We Support You!", "Jean Valjean", "Not Pirates", "Let's Be Friends!", "Alms For The Poor", "REAL KILLERS", "A Single Grain Of Sand", "What's It Matter To You?", "IFF Broken", "Or Something", "Nitroglycerin", "The Ghost of Christmas Present", "God's With Us So You Better Back Off Pal", "Faster Than You", "our shift key is broken", "It Is What It Is", "Bases Loaded", "We Support Law Enforcement", "Cash-Rich, Credit-Poor", "Cunning", "We Never Liked Beauchamp Anyway", "Civilian Ship", "We Didn't Start The Fire, Honest!", "Arson", "Jaywalking", "Bail Bondsmen", "Bail Jumper", "On Parole", "Don't Tell My Probation Officer", "Quit Your Bitching", "And The Then Or Thank You", "Sensor Glitch", "Not My Problem", "Mind Your Business", "Private Vessel Of The Dread Pirate King With His Entire Fleet And Crew In Tow So Back Off", "Daisy", "Sally", "Jane", "Pretty Little Thing", "We Like To Party", "No Squares Here")]"
```

Comment:
it's using the same name list as the freebooter ship, so it shouldn't just be copy-pasted to both

my proposed solution:
add a var to either of the freebooter types like:
```
/obj/effect/overmap/visitable/ship/freebooter_ship
  ...
  var/list/designations = list("Peaceful", "Charming", "Jackal", ...)
```
and then use it in the `New()` proc in both of the ships like:
```
/obj/effect/overmap/visitable/ship/freebooter_salvager/New()
  designation = pick(/obj/effect/overmap/visitable/ship/freebooter_ship::designations)
```